configs/components/nemo/nemo.yaml

```diff
@@ -609,18 +648,41 @@ namelist_changes:
                         nn_rstctl: ${nn_rstctl} 
                         ln_rstart: ${nemo.lresume}
                 namtsd:                                                
-                        ln_tsd_tradmp: ${ln_tsd_tradmp} 
+                        #ln_tsd_tradmp: ${ln_tsd_tradmp} 
                         ln_tsd_init: ${ln_tsd_init} 
-                namdom:
-                        nn_closea: 1
-                        nn_msh: ${nn_msh} 
-                        rn_rdt: ${time_step}
+                #namdom:
+                        #nn_closea: 1
+                        #nn_msh: ${nn_msh} 
+                        #rn_rdt: ${time_step}
                 namsbc:
                         nn_fsbc: ${coupling_freq_in_steps}
                 nammpp:
                         jpni: ${jpni}
                         jpnj: ${jpnj}
-                        jpnij: ${nproc}
+
+choose_generation:
```

Comment:
Is generation a well known term in the NEMO world?

Reply:
I don't know, but we need something to distinguish between 3.6.x and 4.2.x

Reply:
How about version?